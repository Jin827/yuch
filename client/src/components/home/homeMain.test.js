import React from 'react';
import { shallow } from 'enzyme';
import HomeMain from './homeMain';

describe('<HomeMain />', () => {
  const wrapper = shallow(<HomeMain />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('contains images', () => {
    expect(wrapper.find('img').props().src).toBe('text-file-stub');
  });

  it('contains link', () => {
    expect(wrapper.find('a').props().href).toBe(
      'http://www.xn--0z2bz4lsqjfrj.com/',
    );
  });
});
