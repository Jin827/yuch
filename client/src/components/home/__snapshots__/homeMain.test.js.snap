// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeMain /> renders correctly 1`] = `
<div
  className="home-main--container"
  id="home-main"
>
  <div
    className="home-main--box"
  >
    <img
      alt="신선한 음식 배달"
      className="home-main--img"
      src="text-file-stub"
    />
    <div
      className="home-main--text"
    >
      <p
        className="f-mini lh-3"
      >
        OUR RESTAURANT
      </p>
      <h3>
        당신의 위한 유청 서비스
      </h3>
      <div
        className="mb3"
      >
        <h4
          className="b"
        >
          위탁급식
        </h4>
        <p>
          기업계, 각종 행사 모임
          <br />
          연중 중식, 석식, 야식 배달
        </p>
      </div>
      <div>
        <h4
          className="b"
        >
          식당
        </h4>
        <p>
          중식, 석식 뷔폐 서비스
        </p>
      </div>
    </div>
  </div>
  <div
    className="home-main--box reverse"
  >
    <LazyImage
      alt="지역 농산품"
      classes="home-main--img"
      src="text-file-stub"
    />
    <div
      className="home-main--text"
    >
      <h3>
        신선한 지역 농산물
      </h3>
      <p>
        지역 농산물 생산 · 가공 · 소비 활성화를 위하여, 경주 지역민들의 밭에서 공수해온 신선한 농산물들을 주재료로 사용합니다.
      </p>
    </div>
  </div>
  <div
    className="home-main--box"
  >
    <LazyImage
      alt="참기름"
      classes="home-main--img"
      src="text-file-stub"
    />
    <div
      className="home-main--text"
    >
      <h3>
        경주시 지원사업 장려
        <br />
        100% 통참깨 참기름 사용
      </h3>
      <p>
        경주 시니어 클럽에서 주관하는 노인 일자리 창출 사업을 장려하기 위해 
        <a
          className="sesame-link td-none b"
          href="http://www.xn--0z2bz4lsqjfrj.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          착한손맛
        </a>
        의 참기름을 사용합니다.
      </p>
    </div>
  </div>
  <div
    className="home-main--box reverse"
  >
    <LazyImage
      alt="여성을 위한 기업"
      classes="home-main--img"
      src="text-file-stub"
    />
    <div
      className="home-main--text"
    >
      <h3>
        여성이 일하기 좋은 기업
      </h3>
      <p>
        2016년 여성이 일하기 좋은 기업으로 선정.
        <br />
        유청은 나이와 국적에 제한을 두지 않고 채용합니다.
      </p>
    </div>
  </div>
</div>
`;
